content/concepts/payment-system-basics/accounts/reserves.md

```diff
@@ -11,51 +11,58 @@ top_nav_grouping: Popular Pages
 
 The XRP Ledger applies _reserve requirements_, in XRP, to protect the shared global ledger from growing excessively large as the result of spam or malicious usage. The goal is to constrain the growth of the ledger to match improvements in technology so that a current commodity-level machine can always fit the current ledger in RAM.
 
-To have an account, an address must hold a minimum amount of XRP in the shared global ledger. You cannot send this XRP to other addresses. To fund a new address, you must send that address enough XRP to meet the reserve requirement.
+To have an account, an address must hold a minimum amount of XRP in the shared global ledger. To fund a new address, you must receive enough XRP at that address to meet the reserve requirement. You cannot send the reserved XRP to others, but you can recover some of the XRP by [deleting the account](accounts.html#deletion-of-accounts).
 
-The reserve requirement changes from time to time due to the [Fee Voting](fee-voting.html) process, where validators can agree to new reserve settings. On [Mainnet](parallel-networks.html), the current minimum reserve requirement is **10 XRP**. (This is the cost of an address that owns no other objects in the ledger.) Each new [account](accounts.html) must set aside this much XRP. Some of this XRP can be recovered by [deleting the account](accounts.html#deletion-of-accounts).
-
-To determine the current minimum reserve requirement, use the [`server_info` command](server_info.html) and take `validated_ledger.reserve_base_xrp`.
+The reserve requirement changes from time to time due to the [Fee Voting](fee-voting.html) process, where validators can agree to new reserve settings.
 
 ## Base Reserve and Owner Reserve
 
 The reserve requirement has two parts:
 
 * The **Base Reserve** is a minimum amount of XRP that is required for each address in the ledger.
-* The **Owner Reserve** is an increase to the reserve requirement for each object that the address owns in the ledger.
+* The **Owner Reserve** is an increase to the reserve requirement for each object that the address owns in the ledger. The cost per item is also called the _incremental reserve_.
+
+The current reserve requirements on Mainnet are:
+
+- Base reserve: **10 XRP**
+- Owner reserve: **2 XRP** per item
+
+Reserves on other networks may vary.
 
+## Owner Reserves
 
-### Owner Reserves
+Many objects in the ledger (ledger entries) are owned by a particular account. Usually, the owner is the account that created the object. Each object increases the owner's total reserve requirement by the owner reserve. When objects are removed from the ledger, they no longer count against the reserve requirement.
```

Comment:
```suggestion
Many objects in the ledger (ledger entries) are owned by an account--usually the account that created it. Each object owned by an account increases their "owner reserve" requirement. When objects are removed from the ledger, they no longer count against the reserve requirement.
```
When objects are removed does the reserve XRP get added back to the account balance?

Reply:
The balance never changes, but the amount that you're allowed to spend from it does.